assignments/function-conversion.js

```diff
@@ -5,21 +5,41 @@
 // };
 // myFunction();
 
+let myFunction = => () {
+console.log("Function was invoked!");
+};
+myFunction();
+
```

Comment:
This is really close!! You just need to move the parenthesis over to the left like so 
<h3>let myFunction = () => {
console.log("soso")
}</h3>